test/test_grid_manipulation.py

```diff
@@ -71,62 +56,41 @@ def test_build_edge_coordinates():
     grid = two_triangles()
     grid.build_edge_coordinates()
     coords = grid.edge_coordinates
-    # nodes = grid.nodes  # FIXME: Assigned but unused!
-    print(coords)
 
     assert coords.shape == (5, 2)
-    assert np.allclose(coords, [[1.1, 0.1],
-                                [2.6, 1.1],
-                                [2.1, 2.1],
-                                [0.6, 1.1],
-                                [1.6, 1.1]])
+    assert np.allclose(coords, [[1.1, 0.1], [2.6, 1.1],
+                                [2.1, 2.1], [0.6, 1.1], [1.6, 1.1]])
 
 
 def test_build_boundary_coordinates():
     grid = two_triangles()
-    # Add some boundaries.
-    grid.boundaries = [(0, 1),
-                       (0, 2),
-                       (2, 3),
-                       (1, 3)]
+    grid.boundaries = [(0, 1), (0, 2), (2, 3), (1, 3)]
```

Comment:
I guess if it can all fit on one like this is better -- maybe. But often you need to add and remove individual points -- it may get to long for a line, and you can add and remove temporarily with a comment if each logical section has it's own line.